docs/advanced/witness.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 8
+sidebar_label: Witness Node involved Cluster
+title: "Witness Node involved Cluster"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1/advanced/witness/"/>
+</head>
+
+_Available as of v1.3.0_
+
+
+On the edge cases, the witness node plays a specific role in maintaining the cluster's high availability. The witness node is a lightweight node that does not store any data and does not have any workload. This means the witness node can be a lightweight node, which will benefit both cost and high availability.
+
+Harvester now supports the witness node involved cluster. You can enable the witness node when joining the cluster.
+For information about the role management, please refer to [role management](../TBD.md).
```

Comment:
I will update this link after https://github.com/harvester/docs/pull/520 is merged.

Reply:
updated!